lib/http-bridge/src/Cardano/Wallet/HttpBridge/Network.hs

```diff
@@ -100,15 +170,16 @@ mkNetworkLayer httpBridge = NetworkLayer
     , networkTip =
         snd <$> getNetworkTip httpBridge
     , postTx = postSignedTx httpBridge
+    , staticBlockchainParameters = (block0, byronBlockchainParameters @n)
     }
 
 -- | Creates a cardano-http-bridge 'NetworkLayer' using the given connection
 -- settings.
 newNetworkLayer
     :: forall n. KnownNetwork (n :: Network)
-    => Int
+    => PortNumber
```

Comment:
Minor suggestion, but we have a `Port (purpose :: Symbol)` type somewhere.. I think it's in the `cardano-wallet-cli` module. It's quite handy for self-documenting signature, especially because it gets quite confusing quite quickly which port is what (backend API vs our API)